filetypes/yaml.go

```diff
@@ -13,6 +14,17 @@ func FromYamlToMap(f backend.File) (map[string]interface{}, error) {
 		return nil, err
 	}
 
+	// Try to decrypt if the content is SOPS-encrypted
+	if sops.IsEncrypted(bytes) {
+		log.Debug().Str("file", f.Name()).Msg("attempting to decrypt SOPS-encrypted content")
```

Comment:
Logging the error is enough. Don't want this logging every time a config is fetched.